docs/examples.md

```diff
@@ -0,0 +1,35 @@
+# Examples
+
+!!! Tip "Logfire playground"
+Example projects provide options to try out Logfire without needing to instrument your app in production.
```

Comment:
I think this is a weird opening sentence. We want people to use logfire to instrument in production. And there's no reason they can't try out logfire on their app running locally.